Foundation/ProcessInfo.swift

```diff
@@ -41,13 +41,28 @@ open class ProcessInfo: NSObject {
         
     }
     
-    
-    internal static var _environment: [String : String] = {
-        return Dictionary<String, String>._unconditionallyBridgeFromObjectiveC(__CFGetEnvironment()._nsObject)
-    }()
-    
     open var environment: [String : String] {
-        return ProcessInfo._environment
+        let equalSign = Int32(UInt8(ascii: "="))
+        var env: [String : String] = [:]
+        var idx = 0
+        let envp = environ
```

Comment:
Should this use `_CFEnviron()` instead of `environ`?

* [ForSwiftFoundationOnly.h](https://github.com/apple/swift-corelibs-foundation/blob/458977b5bc8911832e3bdacb0af9c2b3bc676e1f/CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h#L300)
* [CFPlatform.c](https://github.com/apple/swift-corelibs-foundation/blob/458977b5bc8911832e3bdacb0af9c2b3bc676e1f/CoreFoundation/Base.subproj/CFPlatform.c#L1364-L1372)

Reply:
Yes I think you are correct, it should also work correctly on Windows then as well. I will update it in a bit.